### ISSUE TITLE
Fresh Install (New Computer)

### ISSUE THREAD

**Issue by Archigos** [tonsky/FiraCode#1352]
TL;DR
Fresh Install of Atom on Windows 11, ligatures do NOT work no matter what I do.

===
I just bought a new computer yesterday and allowed Windows 10 to upgrade to Windows 11 before installing any software/files to the system. I installed the latest Atom (as of this morning), then I downloaded Fira Code 6.2 from here. I installed all the *.tff files with no errors and the font shows up in Microsoft Word correctly so I know they worked.

In Atom, I set the Font Family to Fira Code (even tried FiraCode) and the font is monospaced, however, the ligatures do NOT work no matter what file type I am working with. I've restarted Atom numerous times with the same results. I even attempted to use the old setting in the stylesheet settings for Atom without that working as well.

Any suggestions would be appreciated.

**Comment by mfocko** [comment]
I think ligatures are not enabled by default, but I don't have Atom installed right now, so lemme install it and peek into the settings

**Comment by mfocko** [comment]
For me they are enabled by default, I would suggest removing all Fira Code fonts you have installed and reinstalling. Also restart might help to rebuild font caches (that can be triggered manually on Linux, not sure about Windows).

**Comment by Archigos** [comment]
Rebooting a second time fixed the problem. Thank you
